### ISSUE TITLE
Still active project?

### ISSUE THREAD

**Issue by macmpi** [geeks-r-us/volumio-plugins#14]
@geeks-r-us Volumio 2.527 is now using MPD 0.20.18 which fixes under-run issues, and should improve CD playback with many players.
Any plan to continue development?

**Comment by geeks-r-us** [comment]
There are plans to merge it into the plugin repository. But there is a pending PR in volumio ui.

**Comment by macmpi** [comment]
Thanks for the update, good to know!
Isn't Volumio UI PR for Bluetooth plugin only: I guess CD player plugin does not depend on it, right?

**Comment by geeks-r-us** [comment]
That's right, but for both projects I need to prepare the platform depended setup because they use native functions.

**Comment by macmpi** [comment]
hi @geeks-r-us,

Just wondering if you still plan to submit your work on CD Player and Bluetooth Speaker in Volumio plugin service? 

Thanks for consideration.
